pkgs/tools/typesetting/sile/default.nix

```diff
@@ -59,19 +59,23 @@ stdenv.mkDerivation rec {
     makeWrapper
   ];
   buildInputs = [
+    luaEnv
```

Comment:
I am not sure changing its position here changes anyhting ? I just wanted to say: when you override buildinputs, do it like:
`buildInputs = [ newLuaEnv ] ++ oldBuildInputs` rather than `buildInputs = oldBuildInputs ++ [ newLuaEnv ]`

Reply:
This is what we will do, see [this comment](https://github.com/sile-typesetter/sile/issues/1388#issuecomment-1146580456).